### ISSUE TITLE
Change "TotalFreedom" to "Rcon"

### ISSUE THREAD

**Issue by zD12** [TotalFreedom/TotalFreedomMod#710]
Hi there,
With the automatic unmuting and unfreezing, it says "TotalFreedom - Unmuting <player>".  This is perfectly fine when it comes to TotalFreedom, however, other servers running TotalFreedomMod aren't called TotalFreedom.  Due to this, we should rename to Rcon or something more dynamic.

**Comment by SlicedCaek** [comment]
I would say CONSOLE instead of Rcon because most of people use CONSOLE.

**Comment by VjGamingUK** [comment]
I would say RCON because Multiplay doesn't actually use a full console. Only has Rcon.

**Comment by Wild1145** [comment]
No... You do realize that RCON would be a complete misrepresentation of what is actually happening here? RCON is a system in the minecraft server to send a request and only get the response of that request. So calling it either RCON or CONSOLE would be inaccurate. 

It should be "TotalFreedomMod - bla bla bla" as its the TotalFreedomMod that is doing this, not the console, and not the RCON.

**Comment by zD12** [comment]
As far as I'm aware, it could be CONSOLE or Rcon, but, I'd avoid "TotalFreedomMod".  Perhaps "TFM" so it doesn't spam chat nearly as much?

**Comment by Wild1145** [comment]
@zD12 Both Console and RCON are incorrect though, its not someone on console or RCON doing it... TFM maybe, or TotalFreedomMod, its not that long in the grand scheme of things.

**Comment by looperXD** [comment]
Maybe it could be the server name used in config.yml?

**Comment by AlexFreedomMod** [comment]
Yeah it should be the sever name In config.yml like @looperXD said

**Comment by JeromSar** [comment]
I think @looperXD's idea is the best solution here. Thanks!
Accepted.
